lib/route.coffee

```diff
@@ -207,8 +207,21 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    if statusCode in [401, 403]
+      # This is a simple, but not really adequate way to slow down unauthorized scans of the server.
+      # It is not sufficient because hackers can DDOS across multiple threads and systems.
+      # See https://www.owasp.org/index.php/Blocking_Brute_Force_Attacks#Finding_Other_Countermeasures
+      delayInMilliseconds = 500
```

Comment:
I would change this to `minimumDelayInMilliseconds` for clarity. Otherwise it still feels like a magic number to me.